javascript/javascript1/week1/hw1.js

```diff
@@ -0,0 +1,54 @@
+// A future age calculator 
+
+const yearOfBirth = 1987;
+const yearOfFuture = 2027;
+const age = yearOfFuture - yearOfBirth;
+const output = ` you will be ${age} years old in ${yearOfFuture}`;
+console.log(output);
+
+
+// A dog age calculator
+
+ const dogYearOfBirth = 2017;
+ const dogYearFuture = 2027;
+ const dogYear = dogYearFuture - dogYearOfBirth;
+ let shouldShowResultInDogYears; 
+ if(shouldShowResultInDogYears == true)
```

Comment:
You might have forgotten to assign a value to `shoulShowResultInDogYears` here. At this point, its value is `undefined`. Your `if statement` will only ever run the `else` block because `undefined` is not equal to `true`.